src/stubString.js

```diff
@@ -0,0 +1,19 @@
+import { always } from 'ramda';
+
+/**
+ * A function that returns empty string.
+ *
+ * @func stubString
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.1.0|v2.1.0}
+ * @category Function
+ * @sig ... -> ''
```

Comment:
Hindley-Milner signatures don't use literal types. They explusively use types. It's deal with that

Reply:
@char0n you merged it with wrong signature `@sig ... -> ''`?

Reply:
yeah, fixed it here: https://github.com/char0n/ramda-adjunct/commit/555ae2b30c29ae3cdb98b4dfe6b2c33ab4d4617f